requirements.txt

```diff
@@ -0,0 +1,2 @@
+django
```

Comment:
It is of utmost importance to mention the version of the package being used as not doing so will result in the latest version being picked up during a production/other development build, resulting in issues ranging from minor to potentially application breaking, as the latest version might have significantly changed some methods/ replaced them or deleted them completely, etc.